client/message_plaintext.py

```diff
@@ -0,0 +1,68 @@
+import king_phisher.client.plugins as plugins
+import king_phisher.client.gui_utilities as gui_utilities
+
+try:
+	from bs4 import BeautifulSoup
+except ImportError:
+	has_bs4 = False
+else:
+	has_bs4 = True
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Mike Stringer']
+	classifiers = ['Plugin :: Client :: Email :: Message Plaintext']
+	title = 'Message Plaintext'
+	description = """
+	Parse and include a plaintext version of an email based on the HTML version.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+	options = []
+	req_min_version = '1.10.0'
+	version = '1.0'
+	req_packages = {
+		'bs4' : has_bs4
+	}
+	def initialize(self):
+		mailer_tab = self.application.main_tabs['mailer']
+		self.signal_connect('message-create', self.signal_message_create, gobject=mailer_tab)
+		if not has_bs4:
```

Comment:
This shouldn't be necessary, the client won't initialize a plugin if the required package is missing.